index.html

```diff
@@ -24547,11 +24547,11 @@ <h2>
   <span its-locale-filter-list="ja" lang="ja">変更記録</span>
   </h2>
   <p its-locale-filter-list="en" lang="en">The following changes were made since the previous publication.</p>
-  <p its-locale-filter-list="ja" lang="ja">前回の公開時から以下の変更が行われています。</p>
+  <p its-locale-filter-list="ja" lang="ja">前回の公開時から以下の変更が行われています．</p>
   <ul>
     <li>
-      <p its-locale-filter-list="en" lang="en">Combined and updated 3.3.8 f,g,h into one (3.3.8 f)</p>
-      <p its-locale-filter-list="ja" lang="ja">3.3.8 項目 f,g,h を統合して項目 f 一つとして更新</p>
+      <p its-locale-filter-list="en" lang="en">Merging and simplified <a href="https://w3c.github.io/jlreq/#id237">handling of overhanging ruby text on punctuation marks</a> (former 3.3.8 f,g,h)</p>
```

Comment:
Maybe "merging and simplifying" or "merged and simplified"? Or simply use [imperative mood](https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?id=d37cfe3b5c7ef64aff0459ce2730464120c97f30#n135).

I don't have strong opinion on one way or the other, as long as it's consistent.

(Also, s/former/formerly/, short for "formerly known as".)

Reply:
Also, it might be good to use relative URLs (i.e., removing the https://w3c.github.io/jlreq/ part, so the /TR version won't link to the ED.)

Reply:
Ahhh, yes for both two. Thank you for catching.